contrib/pg_tde/documentation/docs/command-line-tools.md

```diff
@@ -0,0 +1,60 @@
+# Command line tools
+
+The `pg_tde` extension provides new command line tools and modifies some existing tools to work with encrypted WAL and tables.
+
+## pg_tde_change_key_provider
+
+A tool for modifying the configuration of a key provider, possibly also changing its type.
+
+This tool edits the configuration files directly, ignoring permissions or running `postgres` processes.
+
+Its only intended use is to fix servers that can't be started up because of inaccessible key providers. 
+
+For example, you restore from an old backup and the address of the key provider changed in the meantime. Use this tool can to correct the configuration, allowing the server to start up.
+
+Use this tool **only when the server is offline.** To modify the key provider configuration when the server is up and running, use the [`pg_tde_change_key_provider_<type>`](fucntions.md#change-an-existing-provider) SQL functions.
```

Comment:
Now that we have a check maybe we do not need to bold this anymore. Or maybe we still need to bold it in the RC but can remove the bold for the GA.

Reply:
I'd leave it at least for RC